app/controllers/menus_controller.rb

```diff
@@ -16,12 +16,12 @@ def create
             menu_category: params[:menu_category],
             menu_cost: params[:menu_cost],
             menu_description: params[:menu_description],
+            menu_image: params[:menu_image]
```

Comment:
```
def index   
      if Owner.find_by(email: current_owner.email)
        @owner_worker= current_owner.email
      elsif Worker.find_by(email: current_owner.email)
        @owner_worker= current_owner.email
      end  
    end
```


try using define_method

Reply:
can we do the payload construction in a separate method within the private method

```
menu=Menu.new(
            menu_name: params[:menu_name],
            menu_category: params[:menu_category],
            menu_cost: params[:menu_cost],
            menu_description: params[:menu_description],
            menu_image: params[:menu_image]
```